### DISCUSSION TITLE
Find and delete the first image on every page by API

### DISCUSSION THREAD

**Discussion by eurosoll** [pdfcpu/pdfcpu#925] [Q&A]
Hi

I have a situation where i need to remove an image from every page in a document. It is always the first image on the page. 

I have tried using images list but that does not give me coordinates.

What would be the best way to achieve this.

Thanks for the help.

**Comment by hhrutter** [comment]
Unfortunately positioning of images happens within the page content stream.
You would need to parse the content stream.

**Comment by eurosoll** [comment]
Thanks. I have worked out how to find the image from the page content stream. Is there an example anywhere of how to remove an object from the dict?

**Comment by eurosoll** [comment]
I found the solution by using dict.Delete.
